I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 85, in script
    s "Time to check out my phone!"
Exception: Unknown keyword arguments: two_window

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "D:\Renpy\renpy-6.99.11-sdk\renpy\bootstrap.py", line 277, in bootstrap
    renpy.main.main()
  File "D:\Renpy\renpy-6.99.11-sdk\renpy\main.py", line 478, in main
    run(restart)
  File "D:\Renpy\renpy-6.99.11-sdk\renpy\main.py", line 144, in run
    renpy.execution.run_context(True)
  File "D:\Renpy\renpy-6.99.11-sdk\renpy\execution.py", line 734, in run_context
    context.run()
  File "game/script.rpy", line 85, in script
    s "Time to check out my phone!"
  File "D:\Renpy\renpy-6.99.11-sdk\renpy\ast.py", line 603, in execute
    renpy.exports.say(who, what, interact=self.interact)
  File "D:\Renpy\renpy-6.99.11-sdk\renpy\exports.py", line 1135, in say
    who(what, interact=interact)
  File "D:\Renpy\renpy-6.99.11-sdk\renpy\character.py", line 873, in __call__
    self.do_display(who, what, cb_args=self.cb_args, **display_args)
  File "D:\Renpy\renpy-6.99.11-sdk\renpy\character.py", line 719, in do_display
    **display_args)
  File "D:\Renpy\renpy-6.99.11-sdk\renpy\character.py", line 484, in display_say
    what_text = show_function(who, what_string)
  File "D:\Renpy\renpy-6.99.11-sdk\renpy\character.py", line 702, in do_show
    **self.show_args)
  File "D:\Renpy\renpy-6.99.11-sdk\renpy\character.py", line 282, in show_display_say
    return renpy.display.screen.get_widget(screen, "what", layer)
  File "D:\Renpy\renpy-6.99.11-sdk\renpy\display\screen.py", line 1176, in get_widget
    screen.update()
  File "D:\Renpy\renpy-6.99.11-sdk\renpy\display\screen.py", line 573, in update
    self.screen.function(**self.scope)
  File "D:\Renpy\renpy-6.99.11-sdk\renpy\ast.py", line 148, in apply_arguments
    return parameters.apply(args, kwargs, ignore_errors)
  File "D:\Renpy\renpy-6.99.11-sdk\renpy\ast.py", line 136, in apply
    raise Exception("Unknown keyword arguments: %s" % ( ", ".join(values.keys())))
Exception: Unknown keyword arguments: two_window

Windows-8-6.2.9200
Ren'Py 6.99.11.1749
Yuri Jam 1.0
